_includes/footer.html

```diff
@@ -55,6 +55,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/twitter.svg"
             alt="Twitter"
             class="brand-icon"
+	    aria-label="Twitter"
```

Comment:
This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```